src/grt/src/GlobalRouter.cpp

```diff
@@ -713,28 +709,29 @@ void GlobalRouter::initializeNets(std::vector<Net*>& nets)
       }
 
       std::vector<RoutePt> pinsOnGrid;
-      findPins(net, pinsOnGrid);
+      int root_idx;
+      findPins(net, pinsOnGrid, root_idx);
 
       if (pinsOnGrid.size() > 1) {
-        float netAlpha = _alpha;
+        float net_alpha = pdrev_alpha_;
         if (_netsAlpha.find(net->getName()) != _netsAlpha.end()) {
```

Comment:
_netsAlpha should be _nets_alpha (and one of these days you should convert grt to use trailing underscores instead of leading)